### ISSUE TITLE
metadata withb webradio

### ISSUE THREAD

**Issue by balbuze** [volumio/Volumio2#2014]
Webradio send metadata but it is used, it does not allow to get info for artist or album.
I made a small mod in the plugin 'Radioparadise'. Now when flac stream is used, we can access info for artist and album.
The new display is like that :
![image](https://user-images.githubusercontent.com/12850139/98446972-c48acc00-2121-11eb-971f-c7475726c034.png)
I think it could be applied for all radios sending data. Most of the time it is displayed like that : 
![image](https://user-images.githubusercontent.com/12850139/98447027-1a5f7400-2122-11eb-8cc9-e3117152a626.png)
using split (' -') to separate artist and title and re affect it on the required fields should do the trick.
The name of the radio is then displayed on the left.
It keeps the display used with other sources, local, qobuz etc. So it is homogenous.
What do you think?
(I not able to do that though...)

**Comment by volumio** [comment]
We could try, but we shall assume that all webradios shall encode (separated by - ) title, artist and album, which IMHO is not the case...

**Comment by balbuze** [comment]
You are right that this not always the case. Some radio don't even send info. But when something is sent, it is often in the form artist - track. Of course, if it not match required info for metadata, it is not different from what we have today.

**Comment by balbuze** [comment]
Back with this...
I tested HifiberryOs and it displays album art and metadata from radio. Not in volumio. I do think that it should work just by re-arranging where and how data are displayed
![image](https://user-images.githubusercontent.com/12850139/99875428-f5720300-2bef-11eb-8508-a0c81e94fe95.png)

**Comment by pierre-wat** [comment]
Hi ! Do you have any news about this issue ? I'm very interesting about it.

**Comment by BebeMischa** [comment]
Is there any movement on this, or did it just got stored in the freezer?

**Comment by mpede** [comment]
Hey there - found a link to this issue on the Volumio Forum and just want to express my strong interest here as well. Did I imagine or did it work in Volumio 2 or was it just that there were icons that have now disappeared as well. Generally the UX when listening to radio is now rather poor compared to 2. Using metadata properly would be golden...
